src/formatters/phone.js

```diff
@@ -5,9 +5,10 @@ export default function({errors, formatted: value, parsed}) {
 
   if(!isNil(value) && value !== "") {
     // remove all non-digits
-    parsed = parsed.toString().replace(/\D/g, "");
-    formatted = parsed.replace(/(\d{3})(\d{3})(\d{4})/, "($1) $2-$3");
-    if(parsed.length !== 10) {
+    let sanitized = value.toString().replace(/\D/g, "");
```

Comment:
I chose to have a sanitized variable for the others to work off of. It just go to messy trying to perform it inline with everything else.

Reply:
@emeidell Why not have the same parsed and formatted structure here?

Reply:
@emeidell please update this to have matching parsed and formatted structure.

Reply:
We are wanting to keep the formatting that we have its just the parsed that we wanted affected. The formatted and parsed should like the following.
Formatted: (xxx) xxx-xxxx
Parsed: xxx-xxx-xxxx